chat-widget/src/components/reconnectchatpanestateful/interfaces/IReconnectChatPaneStatefulProps.ts

```diff
@@ -4,4 +4,5 @@ export interface IReconnectChatPaneStatefulProps extends IReconnectChatPaneProps
     authClientFunction?: string;
     isReconnectEnabled?: boolean;
     reconnectId?: string;
+    redirectInSameWindow?: boolean;
```

Comment:
Please update dummyProps.ts for this change

Reply:
Will do in a separate PR